### DISCUSSION TITLE
Volume mixer for all sound devices

### DISCUSSION THREAD

**Discussion by axg20202** [Belphemur/SoundSwitch#1568] [Q&A]
Is there a way to get the volume mixer to show all the devices configured in Soundswitch? Currently the Volume Mixer menu item simply opens the legacy windows sndvol mixer, which only shows the default sound output (as selected in Soundswitch) and open apps. There really needs to be a mixer that can show all audio devices.

**Comment by slrgt** [comment]
is something like this what you're looking for? https://eartrumpet.app/
